enterprise/backend/test/metabase_enterprise/serialization/upsert_test.clj

```diff
@@ -80,6 +80,9 @@
                   :visualization_settings (if (= 1 instance-num) {:column_settings {}}
                                                                  {:click_behavior {}}))
 
+    (isa? model Metric)
+    (assoc entity :table_id (mt/id :checkins))
```

Comment:
There's a commit in this PR that removes the default `:table_id` for Metrics when used with-temp. It used `(mt/id :checkins)`. But this default isn't computed only when a value is not supplied, it is always computed and then possible merged over. Using any other db like sample-dataset will error because it cannot find a checkins table.